navbar-service.js

```diff
@@ -28,6 +32,23 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  service.site = {
+    title: config.data.siteTitle,
+    brand: config.data.style.brand,
+    url: '/'
+  };
```

Comment:
I'm not sure how this is accomplished using `vars` alone.  We need different values based on the `tenant` parameter.  How does that work?